hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/AbstractTraceEnricher.java

```diff
@@ -69,7 +70,10 @@ protected void addEnrichedAttribute(Event event, String key, AttributeValue valu
     enrichedAttributes.getAttributeMap().put(key, value);
   }
 
-  protected void addEnrichedAttributes(Event event, List<Pair<String, AttributeValue>> attributes) {
+  protected void addEnrichedAttributes(
+      Event event,
+      List<Pair<String, AttributeValue>> attributes,
+      Map<String, AttributeValue> stringAttributeValueMap) {
```

Comment:
nit: `s/stringAttributeValueMap/attributesToEnrich`

Reply:
Also, you have to iterate on `stringAttributeValueMap` to enrich the key and value in the event

Reply:
+1 currently `stringAttributeValueMap` not used.